datafeeder/src/main/java/org/georchestra/datafeeder/autoconf/GeorchestraEmailFactory.java

```diff
@@ -358,9 +358,9 @@ private String variableToPropertyName(final String varName) {
         return propertyName;
     }
 
-    private Set<String> extractVariableNames(String messageTemplate) {
-        Matcher matcher = Pattern.compile("\\$\\{.*\\}").matcher(messageTemplate);
+    static Set<String> extractVariableNames(String messageTemplate) {
         Set<String> varNames = new TreeSet<>();
+        Matcher matcher = Pattern.compile("\\$(\\{(\\w+\\.)*\\w+\\})").matcher(messageTemplate);
```

Comment:
I'd have expected something more "spring-ish" for templating, but I"m pretty sure you already checked ;-) (thymeleaf wasn't doing the job ?)

Reply:
you're probably right. Often one opts for custom instead of learning a new technology, especially when under pressure.